WBooks/Assets.xcassets/bc_nav_bar.png.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Remember to delete the images that you are not using.

Reply:
But this is the background for the nav bar... or are you talking about modifying the .json?